### PR TITLE
feat: basic tailor CLI

### PR DESCRIPTION
**Pull request by mrcjkb**
I had to re-open a new PR for this, because I can only point mrcjkb/tuxedo-rs/pull/1 into my fork.

### CODE REVIEW

**Comment by mrcjkb** [comment]
NixOS installation works for me.

File: nix/module.nix [hunk 1]
```diff
@@ -16,51 +16,59 @@ in {
...
         };
       };
-
-      systemPackages = mkIf cfg.tailor_gui.enable [
-        pkgs.tailor_gui
-      ];
-    };
-  };
+    }
+    {
```
Comment (benneti) [inline nix/module.nix:67]: I personally would write
```
(mkIf cfg.tailor_gui.enable {
  ....
})
```
which I've seen in nixpkgs modules, I guess the main benefit is, that if you add more stuff under the same toggle the git diffs would be easier to read.
(Same for the cli)

**Review by benneti — APPROVED** [review]
In general the nix stuff looks fine to me, left a little comment.

**Comment by AaronErhardt** [comment]
The CI fails with the latest commit, does this need a fix or it the PR ready to merge?

**Comment by mrcjkb** [comment]
> The CI fails with the latest commit, does this need a fix or it the PR ready to merge?

Should be fixed now.
As tailor_cli isn't excluded from the workspace, it is included in the tailord nix derivation.
I have renamed the tailord derivation (which now contains both tailord and tailor_cli in the same package) to "tuxedo-rs".

If the CI succeeds, this should be ready to be merged.

**Comment by AaronErhardt** [comment]
Thanks!

**Pull request merged**